src/commands/synthetics/tunnel.ts

```diff
@@ -71,6 +72,7 @@ export class Tunnel {
    * keepAlive will return a promise that tracks the state of the tunnel (and reject in case of error)
    */
   public async keepAlive() {
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Why ignoring the error?

If I read the code correctly, the assertion `!this.ws.keepAlive()` can never be false, so I'd suggest we remove it as it will be awaited (and the potential error caught) after.

Reply:
I forgot to add a review comment about this. That was a question I wanted to ask, I was afraid to not understand a hidden functionality in this check. I'll remove it.